python/export_util.py

```diff
@@ -0,0 +1,111 @@
+from dataclasses import dataclass
+from datetime import datetime, date, time, timedelta
+import json as js
+from mage.export_import_util.parameters import Parameter
+import mgp
+
+
+@dataclass
+class Node:
+    id: int
+    labels: list
+    properties: dict
+
+    def get_dict(self) -> dict:
+        return {
+            Parameter.ID.value: self.id,
+            Parameter.LABELS.value: self.labels,
+            Parameter.PROPERTIES.value: self.properties,
+            Parameter.TYPE.value: Parameter.NODE.value,
+        }
+
+
+@dataclass
+class Relationship:
+    end: int
+    id: int
+    label: str
+    properties: dict
+    start: int
+
+    def get_dict(self) -> dict:
+        return {
+            Parameter.END.value: self.end,
+            Parameter.ID.value: self.id,
+            Parameter.LABEL.value: self.label,
+            Parameter.PROPERTIES.value: self.properties,
+            Parameter.START.value: self.start,
+            Parameter.TYPE.value: Parameter.RELATIONSHIP.value,
+        }
+
+
+def convert_to_isoformat(property):
+
+    if isinstance(property, timedelta):
+        return Parameter.DURATION.value + str(property) + ")"
+
+    elif isinstance(property, time):
+        return Parameter.LOCALTIME.value + property.isoformat() + ")"
+
+    elif isinstance(property, datetime):
+        return Parameter.LOCALDATETIME.value + property.isoformat() + ")"
+
+    elif isinstance(property, date):
+        return Parameter.DATE.value + property.isoformat() + ")"
```

Comment:
I left a comment about removing the `(` from property names below. If you're doing so, perhaps replace the relevant part with formatted strings: `f({property.isoformat()})"`.

Reply:
I have the `(` on the properties since for example, I can have a property with `date` value and then treat it like a temporal, while it's just a string with value date.

`node Meeting, properties: { type: "date" }`

Let me know if this makes sense to you.

Reply:
Thanks! Your rationale makes total sense to me, I think I can resolve this point now. :slightly_smiling_face: